lib/dynamic_type/README.md

```diff
@@ -262,3 +262,20 @@ dynamic structs should generally not be implemented as a dictionary. Please refe
 semi-dynamic struct.
 
 # Compilation time
+
+The compilation time and memory usage are exponential with respect to the number of dynamic types.
```

Comment:
Maybe cubic, not exponential

Reply:
Hmm, looks like exponential has the best fit compared to quadratic vs cubic:
![image](https://github.com/NVIDIA/Fuser/assets/1032377/be47bf89-d85b-4cf9-9127-af572a432180)
I think we can conclude it's exponentional.